CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ cmake_dependent_option(USE_SSE "Build with x86 SSE instruction support" ON
   "CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL amd64" OFF)
 option(USE_F16C "Build with x86 F16C instruction support" ON) # autodetects support if ON
 option(USE_LAPACK "Build with lapack support" ON)
+option(USE_LAPACKE_INTERFACE "USE LAPACKE interface for lapack support" OFF)
```

Comment:
What's the motivation for continuing to support `USE_LAPACKE_INTERFACE=0`?

Reply:
we can probably remove this flag. I am using this pr to test the ci for now..

Reply:
Actually there might be a need to keep this flag. So the initial motivation to use the LAPACKE apis is that we want to support ilp64 openblas(cblas+lapacke). As I dig into the ci I think even after enabling large tensor by default we can just use ilp64 openblas on some pipelines (ubuntu cpu gpu) rather than all, because there are many different platform and build combinations to tweak if we choose otherwise. So we can use the flag to keep things how they are there.